### PR TITLE
React and Vue docs

### PR DESCRIPTION
**Pull request by jthoms1**
This PR is to add documentation for Vue and React outside of usage examples. These docs will need to mention that Vue is in Alpha, and React is in Beta.

Introduction
 - What is Ionic Framework? (List Vue and React as supported)

Installation
 - CLI Installation
 - Packages & CDN / Using Ionic in <Vue/React>
 - iOS Setup (Capacitor updates)
 - Android Setup (Capacitor updates)

Building ( Capacitor updates bigger issue)

Navigation
 - React
 - Vue
 - Config (Add usage info)
 - Platform (Add usage info)

Publishing ( Capacitor updates bigger issue)

Note:
This is dependent on some UI changes in the docs to allow us to denote the Framework where content is relevant and allow multiple usage information in some code snippets.

### CODE REVIEW

File: src/components/menu/templates/main.tsx [hunk 1]
```diff
@@ -41,7 +41,8 @@ const items = {
     'Advanced': '/docs/theming/advanced',
   },
   'Navigation': {
-    'Angular': '/docs/navigation/angular'
+    'Angular': '/docs/navigation/angular',
```
Comment (dotNetkow) [inline src/components/menu/templates/main.tsx]: Note for when this is ready - changing urls here to /docs/react/navigation etc...

**Comment by brandyscarney** [comment]
Is this still being added?

**Comment by dotNetkow** [comment]
ah, good catch. Ely just merged in updated [React navigation](https://ionicframework.com/docs/react/navigation) so likely can close. @jthoms1 ?

**Comment by dotNetkow** [comment]
@jthoms1 - believe I can close this, as Ely took over the React docs. Re-open if needed :)

**Pull request closed without merging**